lib/diagnostic.rb

```diff
@@ -90,6 +116,9 @@ def example
   end
 
 at  describe 'DELETE destroy' do
-    # your test(s) here
+  it 'is successful and returns an empty response' do
```

Comment:
Looks like you just missed the actual call to the `destroy` action.

`delete :destroy, id: example.id`